grpc/android-cubic/build.gradle

```diff
@@ -0,0 +1,29 @@
+// Top-level build file where you can add configuration options common to all sub-projects/modules.
+
+buildscript {
+    
+    repositories {
+        google()
+        jcenter()
+        
+    }
+    dependencies {
+        classpath 'com.android.tools.build:gradle:3.6.3'
+        classpath 'com.github.dcendents:android-maven-gradle-plugin:2.1' // Add this line
+        classpath 'com.google.protobuf:protobuf-gradle-plugin:0.8.10'
+    }
+}
```

Comment:
I think there should be empty line above. Or there shouldn't be empty line on the 4-th row.